examples/Huggingface_Transformers/Image_classification_docker/README.md

```diff
@@ -0,0 +1,28 @@
+# HF-torchserve-pipeline
```

Comment:
I'm not sure what this README adds on top of the other one, I'd rather we summarize this part and compress it into the HF section

Reply:
Yeah this would be redundant for the PR too as it would not include the 'deploying on AWS EC2 instance' part and future objectives, would remove this asap.